README.md

```diff
@@ -1,2 +1,145 @@
 # active-analyses
 Active ATLAS analyses
+
```

Comment:
@JCZeng1 All of this info should go in `VBS/dataset.md` &mdash; `README.md` is a shared filed and will get cluttered otherwise.

Reply:
Done